app/assets/javascripts/discourse/lib/lazy-load-images.js.es6

```diff
@@ -68,6 +68,28 @@ function show(image) {
       copyImg.style.height = `${imageData.height}px`;
     }
 
+    if (imageData.width && imageData.height) {
+      let inQuote = false;
+      for (let element = image; element; element = element.parentElement) {
```

Comment:
Does this climb the entire DOM tree up to the root? Maybe we should bail once we reach out of the post body?

Reply:
I merged this for-loop with the previous one and made it stop when getting out of the post body.